language_service/src/protocol.rs

```diff
@@ -33,6 +34,21 @@ pub struct DiagnosticUpdate {
     pub errors: Vec<ErrorKind>,
 }
 
+/// This is a string that represents the interpreter-ready name of the test callable.
+/// i.e. "Main.TestCase". Call it by adding parens to the end, e.g. `Main.TestCase()`
+pub type CallableName = String;
+
+/// A string that represents the originating compilation URI of this callable
+pub type CompilationUri = String;
+
+/// A human readable name that represents the compilation.
+pub type HumanReadableName = String;
+
+#[derive(Debug)]
+pub struct TestCallables {
+    pub callables: Vec<(CompilationUri, CallableName, Location, HumanReadableName)>,
+}
+
```

Comment:
At which point do you admit that just declaring a struct with named fields will be easier? :P